src/components/Renderer/index.tsx

```diff
@@ -166,7 +166,7 @@ const RendererBase = ({
 	);
 	const Widget = getWidget(
 		processedValue,
-		schema.format,
+		schema?.format,
```

Comment:
Can you update `ThemedRendererProps` to denote that `schema` is actually optional and can be undefined please

Reply:
Just realized that this is not making much sense, I'm going to check the examples and fix them

Reply:
This seems to only happen for the examples about materialized and invalid fields.
See: https://github.com/balena-io-modules/rendition/blob/2becdac6dfc642f6e02ec7c5fa5b5c90343c3823/src/components/Renderer/examples.ts#L116-L126
See: https://github.com/balena-io-modules/rendition/blob/2becdac6dfc642f6e02ec7c5fa5b5c90343c3823/src/components/Renderer/examples.ts#L341-L366

Imo the typings are correct, it's just that atm we try to use the Renderer recursively before checking whether the resulting schema is valid. What'st the correct thing to do for the materialized field though is harder to answer.